tracardi/process_engine/action/v1/connectors/discord/push/model/configuration.py

```diff
@@ -1,11 +1,12 @@
 from typing import Optional
 
-from pydantic import AnyHttpUrl
+
+from tracardi.domain.named_entity import NamedEntity
 from tracardi.service.plugin.domain.config import PluginConfig
 
 
 class DiscordWebHookConfiguration(PluginConfig):
-    url: AnyHttpUrl
+    source: NamedEntity
```

Comment:
With new plugins please name resource as resource not source - we used to name it source but no longer.